examples/003_short_fiber_example.py

```diff
@@ -72,6 +72,8 @@
 a11 = field_variables[0]
 a11.plot()
 
+# %%
```

Comment:
The DynamicScraper supports only one output plot per cell, so we simply split the code across multiple cells.